NickvisionTubeConverter.Shared/Controllers/PreferencesViewController.cs

```diff
@@ -26,7 +26,7 @@ internal PreferencesViewController()
     {
         var cultures = CultureInfo.GetCultures(CultureTypes.AllCultures).Where(x => !string.IsNullOrEmpty(x.Name)).ToArray();
         var codes = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
-        codes.UnionWith(cultures.Select(x => x.TwoLetterISOLanguageName));
```

Comment:
should this line stay that way it's two letter, two letter with region and three letter?

Reply:
`cultures` contains both "general" cultures (with name like `en`) and region-specific ones (`en-US`), so there's no need for 2-letter names.

Reply:
so `Names` covers both two letter and two letter with region?

Reply:
yep